packages/engine/paima-funnel/src/paima-l2-processing.ts

```diff
@@ -68,11 +73,11 @@ function decodeEventData(eventData: string): string {
 }
 
 export async function processDataUnit(
-  unit: NonTimerSubmittedData,
+  unit: NonTimerSubmittedData & { senderAddress?: string },
```

Comment:
If the goal of this is to track the contract address, that's now part of the `origin` field since #430

Reply:
Thanks for catching this, this is outdated 🙏  I've removed this address.